providers/base/units/mtd/jobs.pxu

```diff
@@ -2,17 +2,24 @@ id: mtd/check-total-count
 category_id: mtd
 _summary: Check the total MTD count for the platform
```

Comment:
I'm wondering if "for the platform" is required here, but it's not a big deal :)